.github/workflows/ci_tests_legacy.yaml

```
@@ -11,7 +11,7 @@ name: GMT Legacy Tests
 on:
   # push:
   #   branches: [ main ]
-  # pull_request:
+  pull_request:
```

Comment this out before merging.
```suggestion
  # pull_request:
```